src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -298,6 +301,20 @@ export class SavedObjectsService
       .toPromise();
     this.config = new SavedObjectConfig(savedObjectsConfig, savedObjectsMigrationConfig);
 
+    const kibanaConfig = await this.coreContext.configService
+      .atPath<KibanaConfigType>('kibana')
+      .pipe(first())
+      .toPromise();
+
+    deprecations.getRegistry('core').registerDeprecations(
```

Comment:
Not sure which `domainId` we want to use here? `core`? `savedObjects`? WDYT?

Reply:
I'd say `savedObjects`